integration-tests/features/steps/common.py

```diff
@@ -982,7 +982,8 @@ def stack_analysis_check_replaces(json_data, component, version, replaced_by, re
 
 
 @then('I should find that the component {component} version {version} has only one replacement')
-def stack_analysis_check_replaces_count(json_data, component, version):
+@then('I should find that the component {component} version {version} has {expected_replacements:d} replacements')
```

Comment:
Ah  ok. I though that for given (and known requirements.txt) the replacements would be known in advance.